### PR TITLE
Support for Shadow of the Demon Lord

### PR DESCRIPTION
**Pull request by Xacus**
Supports Talents and Spells macros.

### CODE REVIEW

File: item-system/demonlord.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
+mergeObject(CONFIG, {
+  xacus: {
```
Comment (illandril) [inline item-system/demonlord.js:2]:
```suggestion
  illandril: {
```

To keep things consistent, I'd like all CONFIG settings for this module to live under `CONFIG.illandril.hotbarUses`, that way the instructions can be simply...
> If you are a module author, you can add support right in your own module by adding a regular expression to this module's config (`CONFIG.illandril.hotbarUses.macros.{system}`, where `{system}` is one of the supported systems).

... as opposed to needing to know which person added support for each specific system.

File: item-system/demonlord.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
+mergeObject(CONFIG, {
```
Comment (illandril) [inline item-system/demonlord.js:1]:
```suggestion
/*
Support for the Shadow of the Demon Lord system was initially added by Xacus
 */

mergeObject(CONFIG, {
```

Also, add a Credits section to the bottom of the README.md to add credit to yourself for adding this support...
```
# Credits
Support for the Shadow of the Demon Lord system was added by [Xacus](https://github.com/Xacus)
```
(feel free to adjust the wording, link elsewhere, and/or change the credit name to however you'd prefer to be credited)

**Review by illandril — CHANGES_REQUESTED** [review]

File: item-system/demonlord.js (hunk 1 above)
Comment (Xacus) [reply item-system/demonlord.js:2]: Makes absolutely sense.

**Review by illandril — APPROVED** [review]

**Comment by illandril** [comment]
Looks like it should be good. I'll merge these in, give it a quick test, then cut a new release once I have time (hopefully tonight).

**Pull request merged**